pkg/apb/deprovision.go

```diff
@@ -73,3 +87,17 @@ func Deprovision(
 
 	return executionContext.PodName, err
 }
+
+func isNamespaceDeleted(name string, log *logging.Logger) (bool, error) {
+	k8scli, err := clients.Kubernetes(log)
+	if err != nil {
+		return false, err
+	}
+
+	namespace, err := k8scli.CoreV1().Namespaces().Get(name, metav1.GetOptions{})
+	if err != nil {
+		return false, err
+	}
+
+	return namespace == nil || namespace.Status.Phase == "Terminating", nil
```

Comment:
Would prefer to use the [constant](https://godoc.org/k8s.io/api/core/v1#NamespacePhase) to compare the value with.

Reply:
thanks, didn't realize that existed.  Fixed